packages/stark-core/src/logging/logging.module.ts

```diff
@@ -1,17 +1,16 @@
 import { NgModule } from "@angular/core";
 import { ActionReducerMap, StoreModule } from "@ngrx/store";
 
-import { loggingReducer } from "./reducers/index";
+import { loggingReducer, StarkLoggingState } from "./reducers/index";
 import { StarkLoggingActions } from "./actions/index";
 import { StarkLoggingServiceImpl, starkLoggingServiceName } from "./services/index";
-import { StarkLoggingApplicationState } from "../common/store/starkCoreApplicationState";
 
-const reducers: ActionReducerMap<StarkLoggingApplicationState, StarkLoggingActions> = {
-	starkLogging: loggingReducer
+const reducers: ActionReducerMap<StarkLoggingState, StarkLoggingActions> = {
+	logging: loggingReducer
```

Comment:
Just curious, why the change of name?
Does this change the store structure? If so then I'd prefer to keep our prefixes.

Reply:
Well, since we define now a "feature" store for every core module, then what we have in the Application Store is something like this:

```javascript
{
  MainAppState :{ }  // root state
  StarkLogging: {  // feature state => with prefix
    logging: ...   // without prefix otherwise it is too verbose having another "starkLogging" inside
  }
  StarkSession: {  // the same principle here
    session: ...
  }
}
```